### DISCUSSION TITLE
withCount not present on model refresh

### DISCUSSION THREAD

**Discussion by n8udd** [laravel/framework#37575] [Q&A]
I was going to submit a bug, but I wanted to check first: 

I would expect the refresh of a model to refresh the count on a model also, but I'm unsure that this should be the desired behaviour?

`$account = Account::withCount('users')->first();`

`$account->users_count;` `// 6`

`$account->refresh();`

`$account->users_count;` `// NULL`

**Comment by xxdalexx** [comment]
I know this was a while ago, but I can confirm this happens as well.

**Comment by Arkhas** [comment]
Up ?
